tests/c2hs/Bar.chs

```diff
@@ -0,0 +1,4 @@
+module Bar (bar) where
+
+bar :: Int
```

Comment:
It would be good if these tests could actually pull a definition from a header in a C dependency, to confirm that it's wired up properly.

Reply:
Done.